app/src/main/res/values/strings.xml

```diff
@@ -36,40 +36,42 @@
 	<string name="share_s">Share %s</string>
 	<string name="search">Search</string>
 	<string name="search_manga">Search manga</string>
-	<string name="manga_downloading_">Manga downloading…</string>
+	<string name="manga_downloading_">Downloading…</string>
 	<string name="processing_">Processing…</string>
-	<string name="download_complete">Download complete</string>
+	<string name="download_complete">Downloaded</string>
 	<string name="downloads">Downloads</string>
-	<string name="by_name">By name</string>
+	<string name="by_name">Name</string>
 	<string name="popular">Popular</string>
 	<string name="updated">Updated</string>
 	<string name="newest">Newest</string>
-	<string name="by_rating">By rating</string>
+	<string name="by_rating">Rating</string>
 	<string name="all">All</string>
-	<string name="sort_order">Sort order</string>
+	<string name="sort_order">Sorting order</string>
 	<string name="genre">Genre</string>
 	<string name="filter">Filter</string>
 	<string name="theme">Theme</string>
 	<string name="light">Light</string>
 	<string name="dark">Dark</string>
+	<string name="black_dark_theme">Black</string>
+	<string name="black_dark_theme_summary">Uses less power on AMOLED screens</string>
 	<string name="automatic">Automatic</string>
 	<string name="pages">Pages</string>
 	<string name="clear">Clear</string>
-	<string name="text_clear_history_prompt">Do you really want to clear all your reading history? This action cannot be undone.</string>
+	<string name="text_clear_history_prompt">Clear all reading history permanently?</string>
 	<string name="remove">Remove</string>
 	<string name="_s_removed_from_history">\"%s\" removed from history</string>
 	<string name="_s_deleted_from_local_storage">\"%s\" deleted from local storage</string>
-	<string name="wait_for_loading_finish">Wait for the load to finish</string>
+	<string name="wait_for_loading_finish">Waiting for loading to finish…</string>
 	<string name="save_page">Save page</string>
-	<string name="page_saved">Page saved successful</string>
+	<string name="page_saved">Saved</string>
```

Comment:
What saved?

Reply:
The page. Evident from both the above, and the key context.